context-py/src/lib.rs

```diff
@@ -6,6 +6,7 @@ use bundle::{
 };
 use codeowners::{BindingsOwners, CodeOwners};
 use context::{env, junit, meta, repo};
+use prost::Message;
```

Comment:
My eyes might be deceiving me, but is this trait used anywhere?

Reply:
It's necessary for decode to be used.
```
no function or associated item named `decode` found for struct `proto::test_context::test_run::TestResult` in the current scope items from traits can only be used if the trait is in scope
```